docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-mqtt.md

```diff
@@ -13,41 +13,69 @@ labels:
     - enterprise
     - oss
 menuTitle: MQTT
-title: Configure the MQTT notifier for Alerting
+title: Configure MQTT notifications
 weight: 140
+refs:
+  notification-template-examples:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/examples/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/examples/
+  notification-templates:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/
+  configure-contact-points:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/manage-contact-points/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/
 ---
 
-# Configure the MQTT notifier for Alerting
+# Configure MQTT notifications
 
-Use the Grafana Alerting - MQTT integration to send notifications to an MQTT broker when your alerts are firing.
+Use the MQTT integration in contact points to send alert notifications to your MQTT broker.
 
-## Procedure
+## Configure MQTT for a contact point
 
-To configure the MQTT integration for Alerting, complete the following steps.
+To create a contact point with MQTT integration, complete the following steps.
 
-1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
-1. On the **Contact Points** tab, click **+ Add contact point**.
-1. Enter a descriptive name for the contact point.
-1. From the Integration list, select **MQTT**.
+1. Navigate to **Alerts & IRM** -> **Alerting** -> **Contact points**.
+1. Click **+ Add contact point**.
+1. Enter a name for the contact point.
+1. From the **Integration** list, select **MQTT**.
 1. Enter your broker URL in the **Broker URL** field. Supports `tcp`, `ssl`, `mqtt`, `mqtts`, `ws`, `wss` schemes. For example: `tcp://127.0.0.1:1883`.
 1. Enter the MQTT topic name in the **Topic** field.
-1. In **Optional MQTT settings**, specify additional settings for the MQTT integration if needed.
-1. Click **Test** to check that your integration works.
+1. (Optional) Configure [additional settings](#webhook-settings).
```

Comment:
```suggestion
1. (Optional) Configure [additional settings](#optional-settings).
```

I think this went to the wrong anchor originally, I've suggested the one I think is  correct.